pyproject.toml

```
@@ -108,6 +108,7 @@ test = [
 ]
 
 docs = [
+  'atsphinx-mini18n==0.4.0',
```

https://github.com/atsphinx/mini18n/issues/6#issuecomment-2894613028
```suggestion
  'atsphinx-mini18n==0.4.1',
```